runtime-infra/pn-data-monitoring.yaml

```diff
@@ -87,6 +92,18 @@ Resources:
             Condition:
               ArnEquals:
                 aws:PrincipalArn: !Sub "arn:${AWS::Partition}:iam::${PnCoreAwsAccountId}:role/pn-data-monitoring-codebuild-role"
+          - Effect: Allow
+            Action:
+              - "s3:Get*"
+              - "s3:List*"              
+            Resource: 
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

ARN in Resource DataMonitoringBucketPolicy contains hardcoded Partition in ARN or incorrectly placed Pseudo Parameters